stefaniaPopescu26/assignment3/Dictionary.java

```diff
@@ -0,0 +1,56 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Scanner;
+import java.util.Set;
+
+/*
+    I implemented the dictionary with a HashMap, so that it will
+    check if a word is in the dictionary with a O(1) complexity
+    and if a sequence is a prefix with a O(n) complexity.
+ */
+
+public class Dictionary extends HashMap<String, Object>{
```

Comment:
As you already mentioned in your comment: this is not very efficient implementation of the dictionary. Try to aim at complexity of the length of the word or prefix that you are looking up. 
Let me know if you need any help :)